Exercises-2-Basic/src/exercises/Num3.java

```diff
@@ -0,0 +1,26 @@
+package exercises;
+
+import java.util.Scanner;
+
+public class Num3 {
+    /*
+     * Write a Java program that takes 3 numbers as input to calculate and print the
```

Comment:
The description should be above class declaration, meaning just above the line:
`public class Num3 {`

Reply:
Done